enterprise/models.py

```diff
@@ -1608,11 +1608,9 @@ def is_audit_enrollment(self):
 
         :return: Whether the course enrollment mode is of an audit type.
         """
-        course_enrollment_api = EnrollmentApiClient()
-        course_enrollment = course_enrollment_api.get_course_enrollment(
-            self.enterprise_customer_user.username,
-            self.course_id
-        )
+
+        course_enrollment = self.course_enrollment
+
         audit_modes = getattr(settings, 'ENTERPRISE_COURSE_ENROLLMENT_AUDIT_MODES', ['audit', 'honor'])
         return course_enrollment and course_enrollment.get('mode') in audit_modes
```

Comment:
You'll have to switch this to plain 'ole `course_enrollment.mode`, since it's now a model instance and not a dict.